OpenRA.Game/Server/Server.cs

```diff
@@ -169,14 +108,14 @@ public Server(IPEndPoint endpoint, ServerSettings settings, ModData modData)
 
 					foreach (var s in checkRead)
 						if (s == listener.Server) AcceptConnection();
-						else if (PreConns.Count > 0)
+						else if (preConns.Count != 0)
```

Comment:
the line was being changed to preConns anyway, so there's no diff noise. Also, what's complicated about "non-zero"? I call pedantics on this one